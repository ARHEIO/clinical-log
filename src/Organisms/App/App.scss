@import url('https://fonts.googleapis.com/css?family=Open+Sans|Quicksand');

html,
body {
  margin: 0;
  height: 100%;
  background-color: #C6C5B9;
}

.app-container {
  display: flex;
  flex-direction: column;
  width: 100%;
  height: 100vh;

  .app {
    flex: 1 0 auto;
    margin: 0 auto;
    padding: 16px 0;
    width: 100%;

    @media (min-width: 960px) {
      width: 800px;
    }
  }
}
