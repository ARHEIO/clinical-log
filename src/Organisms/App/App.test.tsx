import React from 'react';
import { render } from '@testing-library/react';
import pretty from 'pretty';
import App from './App';

describe('App', () => {
  it('should render', () => {
    const { container } = render(<App />);
    expect(pretty(container.innerHTML)).toMatchSnapshot();
  });
});
