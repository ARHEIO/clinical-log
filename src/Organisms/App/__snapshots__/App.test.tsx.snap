// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
"<div class=\\"app-container\\">
  <header class=\\"app-header\\">
    <div class=\\"app-header_top-bar\\"></div>
    <div class=\\"app-header_bottom-bar\\">
      <div class=\\"container\\">
        <h1>South Yarra Health and Wellbeing Clinic</h1><a class=\\"MuiButtonBase-root MuiButton-root MuiButton-contained log-in-button MuiButton-containedPrimary\\" tabindex=\\"0\\" aria-disabled=\\"false\\" href=\\"#/auth\\"><span class=\\"MuiButton-label\\">LOGIN</span><span class=\\"MuiTouchRipple-root\\"></span></a>
      </div>
    </div>
  </header>
  <section class=\\"app\\">
    <div class=\\"killer-container\\">
      <h2>Newsfeed</h2>
      <div class=\\"survivor loading-spinner-container\\">
        <div class=\\"loading-spinner\\"><svg class=\\"loading-spinner-rear\\">
            <circle cx=\\"50%\\" cy=\\"50%\\" r=\\"18px\\"></circle>
          </svg><svg class=\\"loading-spinner-front\\">
            <circle cx=\\"50%\\" cy=\\"50%\\" r=\\"18px\\"></circle>
          </svg></div>
      </div>
    </div>
  </section>
  <footer class=\\"app-footer\\">
    <p>Made as part of a coding challenge</p>
  </footer>
</div>"
`;
